packages/components/src/Table/Table.css

```diff
@@ -1,10 +1,12 @@
 .table {
   --border-color: var(--color-grey--lighter);
   --header-border-color: var(--color-blue);
-  --horizontal-inset: 0.5rem;
-
+  --horizontal-inset: var(--space-base);
   width: 100%;
+
```

Comment:
We are using box shadow here as a pseudo border. When we use `border-collapse: collapse` it takes away the ability to add a border radius. 

Using box shadow allows us to have the best of both worlds